foundations/javascript_basics/object_basics.md

```diff
@@ -176,6 +176,10 @@ If you feel yourself getting overwhelmed or stuck, don't be afraid to go back an
 
 The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
+- [What is the difference between objects and arrays?](https://javascript.info/object#summary)
+- [How do you access object properties?](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics#bracket_notation)
+- [What is `Array.prototype.map()` useful for?](https://www.youtube.com/watch?v=HB1ZC7czKRs&t=233s)
+- [What is `Array.prototype.reduce()` useful for?](https://youtu.be/HB1ZC7czKRs?t=467)
```

Comment:
These are duplicated items and should be removed

Reply:
done, sorry for the mistake